modules/nextflow/src/main/groovy/nextflow/container/PodmanBuilder.groovy

```diff
@@ -0,0 +1,214 @@
+/*
+ * Copyright 2013-2019, Centre for Genomic Regulation (CRG)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package nextflow.container
+import groovy.transform.CompileStatic
+/**
+ * Helper methods to handle podman containers
+ *
+ * @author Paolo Di Tommaso <paolo.ditommaso@gmail.com>
+ * @author tbugfinder <github@online.ms>
+ */
+@CompileStatic
+class PodmanBuilder extends ContainerBuilder<PodmanBuilder> {
+
+    private boolean sudo
+
+    private boolean remove = true
+
+    private boolean userEmulation
+
+    private String registry
+
+    private String name
+
+    private boolean tty
+
+    private static final String USER_AND_HOME_EMULATION = '-u $(id -u) -e "HOME=${HOME}" -v /etc/passwd:/etc/passwd:ro -v /etc/shadow:/etc/shadow:ro -v /etc/group:/etc/group:ro -v $HOME:$HOME'
```

Comment:
This option is not really used. Make sense remove from this implementation.

Reply:
please excuse, `tty` or `USER_AND_HOME_EMULATION`?

Reply:
removed USER_AND_HOME_EMULATION with
commit 13b0fe2072ffb3202d3d0832ceebf4f991f7944a

Reply:
Nice. Also `tty` can go, I think was never really used with docker.

Reply:
removed tty with commit
commit cac990538e098d72152d2fbfdadd019c403b310a

Reply:
removal of userEmulation was reverted
commit 3efc3b9c11c904512676cbb6f5f57ac7397c4243